config/database.yml

```diff
@@ -8,7 +8,9 @@ default: &default
   adapter: postgresql
   pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
   timeout: 5000
-
+  username: postgres
+  password: 123456
+  
 development:
```

Comment:
Everyone can see your credentials, check this, if the .gitignore is correct. Also, try to create a new branch when the previous is already merged.